helpers/initOnboard.ts

```diff
@@ -8,8 +8,8 @@ const injected = injectedModule();
 const walletConnect = walletConnectModule();
 const gnosis = gnosisModule();
 
-const blocknativeDappId = process.env.NEXT_PUBLIC_BLOCKNATIVE_DAPP_ID;
-const infuraId = process.env.NEXT_PUBLIC_INFURA_ID;
+const blocknativeDappId = process.env.NEXT_PUBLIC_BLOCKNATIVE_DAPP_ID ?? "";
+const infuraId = process.env.NEXT_PUBLIC_INFURA_ID ?? "";
```

Comment:
if these are required for the app to run, id rather assert they exist and have the app fail with an error directing the dev to add the right envs. this makes it a lot easier to debug.  if these arent required then u can ignore this comment